socorro/processor/mozilla_transform_rules.py

```diff
@@ -857,10 +858,17 @@ def __init__(self, config):
         try:
             sentry_dsn = self.config.sentry.dsn
         except KeyError:
-            # DotDict raises a KeyError when things are missing
+            # NOTE(willkg): DotDict raises a KeyError when things are missing
             sentry_dsn = None
-
-        self.generator = SignatureGenerator(sentry_dsn=sentry_dsn)
+        self.sentry_dsn = sentry_dsn
+        self.generator = SignatureGenerator(error_handler=self._error_handler)
+
+    def _error_handler(self, raw_crash, processed_crash, exc_info, extra):
+        """Captures errors from signature generation"""
+        extra['uuid'] = raw_crash.get('uuid', None)
```

Comment:
Does `.get` not default to `None`? Otherwise, is that to be more explicit?

Reply:
It depends on what `raw_crash` is. If it's a dict, then yes--it defaults to giving you `None`. If it's not, then who knows. In this case, I'm pretty sure it can be one of a variety of DotDict implementations.